packages/database-mongo/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@accounts/mongo",
+  "version": "0.1.0-beta.3",
+  "description": "MongoDB adaptor for accounts",
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "start": "tsc --watch",
+    "compile": "tsc",
+    "testonly": "jest",
+    "test:watch": "jest --watch",
+    "coverage": "yarn testonly --coverage",
+    "prepublishOnly": "yarn compile"
+  },
+  "renovate": {
```

Comment:
can you remove renovate config also please?

Reply:
also done, along with the circleci folder and the .gitignore